webapp/app.py

```diff
@@ -152,3 +153,19 @@ def site_map_operators():
     response.headers["Content-Type"] = "application/xml"
 
     return response
+
+
+def schedule_banner(start_date: str, end_date: str):
+    try:
+        end = datetime.strptime(end_date, "%Y-%m-%d")
+        start = datetime.strptime(start_date, "%Y-%m-%d")
+        present = datetime.now()
+        print("hello", start, present)
+        return start <= present < end
+    except ValueError:
+        return False
+
+
+@app.context_processor
+def utility_processor():
```

Comment:
Can you move these into `handlers.py`, where `utility_processor` is already defined? I'd like to keep `app.py` as clean as possible, thanks.